app/models/spree/line_item_decorator.rb

```diff
@@ -56,6 +57,10 @@ def included_tax
     adjustments.included_tax.sum(&:included_tax)
   end
 
+  def tax_rates
+    product.tax_category.tax_rates
```

Comment:
Not every product has a tax_category, so we will need to handle the nil case here:

```ruby
product.tax_category.andand.tax_rates || []
```